tests/src/Factory/Api/Twitter/DirectMessageTest.php

```diff
@@ -67,7 +67,7 @@ public function testApiFormatMessagesWithHtmlText()
 			->toArray();
 
 		self::assertEquals('item_title', $directMessage['title']);
-		self::assertEquals('<strong>item_body</strong>', $directMessage['text']);
+		self::assertEquals('<b>item_body</b>', $directMessage['text']);
```

Comment:
There is now a dedicated "strong" element, so there is "b" *and* "strong".

Reply:
Why do we need both? Do we discriminate between both in our display? Do we expect remote platforms to discriminate between both? I'd rather we keep it simpler.

Reply:
`b` and `strong` are different elements and neither of them is deprecated, see here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element

They could be treated differently by for example screenreaders. It's the same like just using bigger font sizes instead of using some header elements like `h1`, `h2`, ... Optically it looks the same, but technically it is different.

Reply:
I didn't say any of them was deprecated, I'm saying `<strong>` isn't relevant for us because our post body editor isn't semantic-centric, it's formatting-centric. So we should stick to `<b>`.

Reply:
We can define headlines, which definitely is semantic and not formatting. But anyway: I don't see a reason not to expand the capabilities here.